modules/aspect_bazel_lib/0.3.0/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+build_targets: &build_targets
```

Comment:
Should this be "test_targets"?

Reply:
that would be nice, but our tests can't pass here yet because of various errors. One is local_repository so I'll try again with https://groups.google.com/g/bazel-discuss/c/xpsg3mWQPZg

Reply:
https://github.com/bazelbuild/bazel-central-registry/pull/42 is where I'm trying to change it to run tests, still failing